cpp/src/mip/local_search/local_search.cu

```diff
@@ -26,10 +26,86 @@
 #include <utilities/seed_generator.cuh>
 #include <utilities/timer.hpp>
 
+#include <mip/feasibility_jump/fj_cpu.cuh>
+
 #include <cuda_profiler_api.h>
 
+#include <future>
+
 namespace cuopt::linear_programming::detail {
 
+template <typename i_t, typename f_t>
+cpu_fj_thread_t<i_t, f_t>::cpu_fj_thread_t()
+{
+  cpu_worker = std::thread(&cpu_fj_thread_t<i_t, f_t>::cpu_worker_thread, this);
```

Comment:
Nicolas switched everything to using openmp. 

Any thoughts on using one or the other. I am worried of different approaches here.